docs/dev/developer-guides/aa.md

```diff
@@ -77,7 +77,7 @@ Like in EIP4337, our account abstraction protocol supports paymasters: accounts
 Each paymaster should implement the [IPaymaster](https://github.com/matter-labs/v2-testnet-contracts/blob/main/l2/system-contracts/interfaces/IPaymaster.sol) interface. It contains the following two methods:
 
 - `validateAndPayForPaymasterTransaction` is mandatory and will be used by the system to determine if the paymaster approves paying for this transaction. If the paymaster is willing to pay for the transaction, this method must send at least `tx.gasprice * tx.gasLimit` to the operator. It should return the `context` that will be one of the call parameters to the `postTransaction` method.
-- `postTransaction` is optional and will be called after the transaction has been executed. Note that unlike EIP4337, there _is no guarantee that this method will be called_. In particular, this method won't be called if the transaction has failed with `out of gas` error. It takes four parameters: the context returned by the `validateAndPayForPaymasterTransaction` method, the transaction itself, whether the execution of the transaction succeeded, and also the maximum amount of gas the paymaster might be refunded with. More documentation on refunds will be available once their support is added to zkSync.
+- `postTransaction` is optional and is called after the transaction executes. Note that unlike EIP4337, there _is no guarantee that this method will be called_. In particular, this method won't be called if the transaction fails with `out of gas` error. It takes four parameters: the context returned by the `validateAndPayForPaymasterTransaction` method, the transaction itself, whether the execution of the transaction succeeded, and also the maximum amount of gas the paymaster might be refunded with.
```

Comment:
```suggestion
- `postTransaction` is optional and is called after the transaction executes. Note that unlike EIP4337, there _is no guarantee that this method will be called_. In particular, this method won't be called if the transaction fails with `out of gas` error. The method takes four parameters: the context returned by `validateAndPayForPaymasterTransaction`, the transaction itself, a flag that indicates whether the transaction execution succeeded, and the maximum amount of gas the paymaster might be refunded with.

Reply:
this method is already referred to, so It is fine